yapapi/executor/__init__.py

```diff
@@ -427,18 +431,15 @@ async def worker_starter() -> None:
         find_offers_task = loop.create_task(find_offers())
         process_invoices_job = loop.create_task(process_invoices())
         wait_until_done = loop.create_task(work_queue.wait_until_done())
+        worker_starter_task = loop.create_task(worker_starter())
```

Comment:
the name could have been more fancy ;) ... but yeah, I get it - it's actually the `worker_starter` itself that sounds strange ;)